server/src/main/resources/db/changelog/20170301083925-add-cert-serial-revoked-column.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20170301083925-1" author="mstead">
+        <comment>
+            Add cp_cert_serial.revoked column to support removing
+            the hibernate formula off of the CertificateSerial entity.
+        </comment>
+        <addColumn tableName="cp_cert_serial">
+            <column name="revoked" type="BOOLEAN" defaultValue="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="20170301083925-2" dbms="postgresql,oracle" author="mstead">
```

Comment:
@mstead That is very good information to have. We can upgrade to version 3 (which has the proper dbms support on the sql tag as it has already been tagged fro the appropriate repositories.